build.gradle

```diff
@@ -495,6 +495,56 @@ task gatkDoc(type: Javadoc, dependsOn: classes) {
     options.addStringOption("verbose")
 }
 
+// Generate GATK Bash Tab Completion File
+task gatkTabComplete(type: Javadoc, dependsOn: classes) {
+    final File tabCompletionDir = new File("build/docs/tabCompletion")
```

Comment:
We should extract a docRoot variable for the value "build/docs" since its repeated in a few places.